README.md

```diff
@@ -31,8 +31,9 @@ ruby myapp.rb
 View at: [http://localhost:4567](http://localhost:4567)
 
 The code you changed will not take effect until you restart the server.
-Please restart the server every time you change or use
-[sinatra/reloader](http://www.sinatrarb.com/contrib/reloader).
+Please restart the server every time you change or use a code reloader
+like [rerun](https://github.com/alexch/rerun) or
+[rack-unreloader](https://github.com/jeremyevans/rack-unreloader).
```

Comment:
Perhaps it can be mentioned that somenweb servers have this capability? @ioquatix, perhaps you have a wording that'd help a reader of this readme? "Your web server may support code reloading, too." Keeping the tone general.

Reply:
Maybe I not reading a few lines down, where Puma's mentioned. Does that have anything to do with reloading capabilities?

Reply:
@olleolleolle that's a good question, I'm not sure. I feel like that line about Puma could be deleted. I do like the idea of mentioning servers that support code reloading. For Puma I think something like Guard would be required.